manifests/nodejs.yml

```diff
@@ -91,87 +91,189 @@ tests/:
             express4: v3.19.0
     waf/:
       test_addresses.py:
-        Test_BodyJson: v2.2.0
+        Test_BodyJson: 
+          '*': v2.2.0
+          nextjs: v4.17.0
         Test_BodyRaw: missing_feature
-        Test_BodyUrlEncoded: v2.2.0
-        Test_BodyXml: v2.2.0
+        Test_BodyUrlEncoded: 
+            '*': v2.2.0
+            nextjs: missing_feature  # request.formData() method not supported yet
+        Test_BodyXml: 
+            '*': v2.2.0
+            nextjs: irrelevant  # Body xml is not converted to JSON in nextjs
         Test_ClientIP: missing_feature
-        Test_Cookies: v2.0.0
+        Test_Cookies:
+          '*': v2.0.0
+          nextjs: v4.17.0
         Test_FullGrpc: missing_feature
-        Test_GraphQL: v4.16.0
-        Test_Headers: v2.0.0
+        Test_GraphQL:
+          '*': v4.16.0
+          nextjs: irrelevant # nextjs is not related with graphql
+        Test_Headers:
+          '*': v2.0.0
+          nextjs: v4.17.0
```

Comment:
Aren't our headers protection framework independant ?

Reply:
same thing for status code, raw url, etc...

Reply:
I had some doubts here. To add version or not... yes headers/url... features are independent, but "officially" nextjs is not supported until 4.17.0...
But you're right, is should work so I'll remove the nextjs specific version to these tests.